scripts/update-docs.sh

```diff
@@ -1,11 +1,11 @@
 #!/bin/sh
 
 # convert doxygen xml to markdown for usage with hugo static site gen
-./convert-doxygen-to-md.sh
+./convert-doxygen-to-md.sh && cd ..
 
 # generate site
-cd ../misc/docs-hugo && hugo && cd ../../scripts
+cd misc/docs-hugo && hugo && cd ../..
 
 # remove existing docs/ and add new in its place
-rm -rf ../docs/* && cp -r ../misc/docs-hugo/public/* ../docs/
+rm -rf ../docs/* && cp -r misc/docs-hugo/public/* ../docs/
```

Comment:
It looks like the path got messed up. As a result, you deleted the `docs/` folder so it's no longer on this branch

```suggestion
rm -rf docs/* && cp -r misc/docs-hugo/public/* docs/
```